### ISSUE TITLE
Error when running on Win10

### ISSUE THREAD

**Issue by Hagtronics** [chipmuenk/pyfda#214]
Windows 10 / 21H2
Python 3.9.9
All libraries listed installed and updated as of today,
    PyQt / Qt5
    numpy
    numexpr
    scipy: 1.2.0 or higher
    matplotlib: 3.1 or higher
    Markdown

pydfa Installed as of today.
Try to run it and I get these errors,

===========================

C:\Users\steve>pyfdax
KeyError: 'V_NMG'
[   INFO] [pyfda.pyfda_rc:187] Using 'DejaVu Sans' font.
[   INFO] [pyfda.libs.tree_builder:256] Parsing config file
        'C:\Users\steve\.pyfda\pyfda.conf' with sections:
                [Common]
                [Input Widgets]
                [Plot Widgets]
                [Filter Widgets]
                [Fixpoint Widgets]

[   INFO] [pyfda.libs.tree_builder:310] Found 3 entries in [Common]
[   INFO] [pyfda.libs.tree_builder:336] No valid user directory specified.
[   INFO] [pyfda.libs.tree_builder:594] Found 5 classes in [Input Widgets]:
        pyfda.input_widgets.input_specs.Input_Specs
        pyfda.input_widgets.input_coeffs.Input_Coeffs
        pyfda.input_widgets.input_pz.Input_PZ
        pyfda.input_widgets.input_info.Input_Info
        pyfda.input_widgets.input_fixpoint_specs.Input_Fixpoint_Specs

[   INFO] [pyfda.libs.tree_builder:594] Found 6 classes in [Plot Widgets]:
        pyfda.plot_widgets.plot_hf.Plot_Hf
        pyfda.plot_widgets.plot_phi.Plot_Phi
        pyfda.plot_widgets.plot_tau_g.Plot_tau_g
        pyfda.plot_widgets.plot_pz.Plot_PZ
        pyfda.plot_widgets.plot_impz.Plot_Impz
        pyfda.plot_widgets.plot_3d.Plot_3D

[   INFO] [pyfda.libs.tree_builder:594] Found 10 classes in [Filter Widgets]:
        pyfda.filter_widgets.bessel.Bessel
        pyfda.filter_widgets.butter.Butter
        pyfda.filter_widgets.cheby1.Cheby1
        pyfda.filter_widgets.cheby2.Cheby2
        pyfda.filter_widgets.ellip.Ellip
        pyfda.filter_widgets.equiripple.Equiripple
        pyfda.filter_widgets.firwin.Firwin
        pyfda.filter_widgets.ma.MA
        pyfda.filter_widgets.manual.Manual_FIR
        pyfda.filter_widgets.manual.Manual_IIR

[   INFO] [pyfda.libs.tree_builder:594] Found 1 classes in [Fixpoint Widgets]:
        pyfda.fixpoint_widgets.fir_df.fir_df_pyfixp_ui.FIR_DF_pyfixp_UI

[   INFO] [pyfda.input_widgets.input_specs:322] Start filter design using method
        'Cheby1.LPman'
[   INFO] [pyfda.input_widgets.input_specs:358] Designed filter with order = 10
[WARNING] [pyfda.input_widgets.input_fixpoint_specs:86] No fixpoint filter found for this type of filter!
Traceback (most recent call last):
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 197, in _run_module_as_main
    return _run_code(code, main_globals, None,
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\Scripts\pyfdax.exe\__main__.py", line 7, in <module>
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\pyfdax.py", line 294, in main
    mainw = pyFDA()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\pyfdax.py", line 143, in __init__
    self._construct_UI()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\pyfdax.py", line 156, in _construct_UI
    pltTabWidgets = plot_tab_widgets.PlotTabWidgets(self)  # plot widgets
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tab_widgets.py", line 36, in __init__
    self._construct_UI()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tab_widgets.py", line 74, in _construct_UI
    inst = wdg_class()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tau_g.py", line 60, in __init__
    self._construct_UI()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tau_g.py", line 96, in _construct_UI
    self.draw()  # initial drawing of tau_g
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tau_g.py", line 164, in draw
    self.calc_tau_g()
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\plot_widgets\plot_tau_g.py", line 148, in calc_tau_g
    self.W, self.tau_g = group_delay(fb.fil[0]['sos'], nfft=params['N_FFT'],
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\libs\pyfda_sig_lib.py", line 618, in group_delay
    w, tau_g = sos_group_delayz(b, w, fs=fs)
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\pyfda\libs\pyfda_sig_lib.py", line 708, in sos_group_delayz
    sos, n_sections = sig.filter_design._validate_sos(sos)
  File "C:\Users\steve\AppData\Local\Programs\Python\Python39\lib\site-packages\scipy\signal\filter_design.py", line 34, in __getattr__
    raise AttributeError(
AttributeError: scipy.signal.filter_design is deprecated and has no attribute _validate_sos. Try looking in scipy.signal instead.

=====================

Not sure what to try next. Any ideas?

I appreciate your time and help.

**Comment by chipmuenk** [comment]
Which scipy version are you using? It seems I have used a deprecated function that is no longer supported in your scipy version. Please try to downgrade your scipy version and tell me whether that works for you. I'll replace the function in my next release.

**Comment by chipmuenk** [comment]
Thanks, I'll leave the issue open as a reminder.

**Comment by karatemir** [comment]
@chipmuenk same here, fresh venv on ubuntu 20.04 and `pip install pyfda` installed with it scipy 1.8.0. Downgrading to 1.7.3 did the trick.

**Comment by g-berthiaume** [comment]
Hi
I have the exact same issue, also on a windows machine.

Here's some system info
```
λ  python --version
Python 3.10.2

λ  python -m pip list | grep scipy
scipy               1.8.0

λ  python -m pip list | grep pyfda
pyfda               0.6.0
```

**Comment by chipmuenk** [comment]
Fixed in v0.6.1

**Comment by Hagtronics** [comment]
Checked install today with both versions of Scipy (1.7.3 and current 1.8.0) and it works for me. Thanks for the unbelievably useful program.
